Parsers/ASimAuthentication/Tests/Microsoft_Github_ASimAuthentication_SchemaTest.csv

```diff
@@ -0,0 +1,99 @@
+"(2) Info: Missing optional field [Rule]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: Missing optional field [SrcDeviceType]"
+"(2) Info: Missing optional field [SrcDomain]"
+"(2) Info: Missing optional field [SrcDvcId]"
+"(2) Info: Missing optional field [SrcDvcOs]"
+"(2) Info: Missing optional field [SrcDvcScopeId]"
+"(2) Info: Missing optional field [SrcDvcScope]"
+"(2) Info: Missing optional field [SrcFQDN]"
+"(2) Info: Missing optional field [SrcGeoCity]"
+"(2) Info: Missing optional field [SrcGeoLatitude]"
+"(2) Info: Missing optional field [SrcGeoLongitude]"
+"(2) Info: Missing optional field [SrcGeoRegion]"
+"(2) Info: Missing optional field [SrcHostname]"
+"(2) Info: Missing optional field [SrcIsp]"
+"(2) Info: Missing optional field [SrcOriginalRiskLevel]"
+"(2) Info: Missing optional field [SrcPortNumber]"
+"(2) Info: Missing optional field [SrcRiskLevel]"
+"(2) Info: Missing optional field [TargetAppId]"
+"(2) Info: Missing optional field [TargetDescription]"
+"(2) Info: Missing optional field [TargetDeviceType]"
+"(2) Info: Missing optional field [TargetDvcId]"
+"(2) Info: Missing optional field [TargetDvcOs]"
+"(2) Info: Missing optional field [TargetDvcScopeId]"
+"(2) Info: Missing optional field [TargetDvcScope]"
+"(2) Info: Missing optional field [TargetFQDN]"
+"(2) Info: Missing optional field [TargetGeoCity]"
+"(2) Info: Missing optional field [TargetGeoCountry]"
+"(2) Info: Missing optional field [TargetGeoLatitude]"
+"(2) Info: Missing optional field [TargetGeoLongitude]"
+"(2) Info: Missing optional field [TargetGeoRegion]"
+"(2) Info: Missing optional field [TargetIpAddr]"
+"(2) Info: Missing optional field [TargetOriginalAppType]"
+"(2) Info: Missing optional field [TargetOriginalRiskLevel]"
+"(2) Info: Missing optional field [TargetOriginalUserType]"
+"(2) Info: Missing optional field [TargetPortNumber]"
+"(2) Info: Missing optional field [TargetRiskLevel]"
+"(2) Info: Missing optional field [TargetSessionId]"
+"(2) Info: Missing optional field [TargetUrl]"
+"(2) Info: Missing optional field [TargetUserId]"
+"(2) Info: Missing optional field [TargetUserScopeId]"
+"(2) Info: Missing optional field [TargetUserScope]"
+"(2) Info: Missing optional field [TargetUserType]"
+"(2) Info: Missing optional field [ThreatCategory]"
+"(2) Info: Missing optional field [ThreatConfidence]"
+"(2) Info: Missing optional field [ThreatField]"
+"(2) Info: Missing optional field [ThreatFirstReportedTime]"
+"(2) Info: Missing optional field [ThreatId]"
+"(2) Info: Missing optional field [ThreatIpAddr]"
+"(2) Info: Missing optional field [ThreatIsActive]"
+"(2) Info: Missing optional field [ThreatLastReportedTime]"
+"(2) Info: Missing optional field [ThreatName]"
+"(2) Info: Missing optional field [ThreatOriginalConfidence]"
+"(2) Info: Missing optional field [ThreatOriginalRiskLevel]"
+"(2) Info: Missing optional field [ThreatRiskLevel]"
+"(2) Info: Missing optional field [User]"
```

Comment:
please map TargetUsername with User

Reply:
Done

Reply:
still missing in vim file

Reply:
Updated

Reply:
@gokuldkate missing in vim file. please add https://github.com/Azure/Azure-Sentinel/blob/7c544459a874046e9ed686f7a9c81b3b91993a32/Parsers/ASimAuthentication/Parsers/vimAuthenticationGithub.yaml